client/reader/search-stream/index.jsx

```diff
@@ -235,9 +235,11 @@ class SearchStream extends Component {
 							delayTimeout={ 500 }
 							placeholder={ searchPlaceholderText } />
 					</CompactCard>
-					<p className="search-stream__blank-suggestions">
-						{ this.props.translate( 'Suggestions: {{suggestions /}}.', { components: { suggestions: sugList } } ) }
-					</p>
+					{ this.state.suggestions &&
```

Comment:
then it wouldn't be any different :)
Its slightly more defensive but since you looked into how `state.suggessions` is being populated we are all good!